lib/membrane_rtc_engine/endpoints/webrtc/track_receiver.ex

```diff
@@ -177,6 +180,11 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackReceiver do
     {{:ok, maybe_request_keyframe(state.selector.current_variant)}, state}
   end
 
+  @impl true
+  def handle_event(_pad, %VoiceActivityChanged{} = event, _ctx, state) do
+    {{:ok, notify: event}, state}
```

Comment:
We should extend `notifications()` type and, to be consistent with how we notify about TrackVariantSwitched, I would change the notification to `{:voice_activity_changed, status}`. I'm sorry for not mentioning this in the first round of CR :slightly_frowning_face: